package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "macOSNotifJS",
+  "version": "0.0.1",
+  "description": "![macOsNotifJS](brand/macOSNotifJS-408x80.png)\\ **A simple Javascript plugin to create simulated macOS notifications on your website.**",
+  "main": "src/macOSNotif.js",
+  "scripts": {
+    "build": "webpack && npm run minifycss",
+    "minifycss": "cleancss -o ./dist/macOSNotif.min.css ./dist/macOSNotif.css "
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MattIPv4/macOSNotifJS.git"
+  },
+  "keywords": [],
+  "author": "MattIPv4",
+  "license": "ISC",
```

Comment:
Should be `AGPL-3.0-only`?

Reply:
Yes, you are right. Didn't notice it (was auto generated by npm). I will update this PR later today